components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/listener/SCIMUserOperationListener.java

```diff
@@ -167,6 +169,11 @@ public boolean doPostSetUserClaimValueWithID(String s, UserStoreManager userStor
     public boolean doPreSetUserClaimValueWithID(String userID, String claimURI, String claimValue, String profileName,
                                                 UserStoreManager userStoreManager) throws UserStoreException {
 
+        // Validate whether claim update request is for a provisioned user.
+        if (isJitProvisionEnhancedFeature()) {
```

Comment:
isn't this check inside "validateJitProvisionedUserClaimUpdate"

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/367/commits/615427cb98805016d37d6068432a2f558703d653